docs/install.md

```diff
@@ -61,13 +62,39 @@ Actually we use vanilla free5gc::docker-compose with some overrides(see docker-c
 
 So you can clone free5gc docker-compose and test free5gc upf, then add edgecom override files and feel the differences.
 
-0. pull repository: `git clone https://github.com/edgecomllc/free5gc-compose.git`
+0. Pull repository: `git clone https://github.com/edgecomllc/free5gc-compose.git`
 0. Run containers based on docker hub images:
    ```bash
    cd free5gc-compose
    docker compose pull
    docker compose up -d
    ```
+0. Check if UPF and SMF containers are running correctly:
```

Comment:
`Correctly` means that successful association setup should be stated in both logs